cgi-bin/DW/Console/Command/ScreenSet.pm

```diff
@@ -1,25 +1,31 @@
-# This code was forked from the LiveJournal project owned and operated
-# by Live Journal, Inc. The code has been modified and expanded by
-# Dreamwidth Studios, LLC. These files were originally licensed under
-# the terms of the license supplied by Live Journal, Inc, which can
-# currently be found at:
+#!/usr/bin/perl
 #
-# http://code.livejournal.org/trac/livejournal/browser/trunk/LICENSE-LiveJournal.txt
+# DW::Console::Command::ScreenSet
+#
+# Console commands for setting and clearing suspend notes. If a
```

Comment:
*facepalm* copied the format from another file and didn't finish changing the content. right.